microk8s-resources/actions/common/utils.sh

```diff
@@ -470,7 +470,8 @@ render_csr_conf() {
     if ! [ "$IP_ADDRESSES" == "127.0.0.1" ] && ! [ "$IP_ADDRESSES" == "none" ]
     then
         local ips='' sep=''
-        local -i i=3
+        local ip_count=$(grep -ce '^IP' certs/csr.conf.template)
+        local -i i=$((ip_count+1))
         for IP_ADDR in $(echo "$IP_ADDRESSES"); do
             ips+="${sep}IP.$((i++)) = ${IP_ADDR}"
```

Comment:
Continuing on the previous comment, I think it is better to add `microk8s-` (or similar) prefix to the IP addresses added automatically by MicroK8s, something like this:

```suggestion
            ips+="${sep}IP.microk8s-$((i++)) = ${IP_ADDR}"
```

Note that we also do something similar already for the MicroK8s charm, see https://github.com/canonical/charm-microk8s/blob/master/config.yaml#L98-L99